membership-attribute-service/app/services/ScanamoAttributeService.scala

```diff
@@ -8,14 +8,16 @@ import com.gu.scanamo.syntax.{set => scanamoSet, _}
 import com.gu.scanamo.update.UpdateExpression
 import com.typesafe.scalalogging.LazyLogging
 import models.Attributes
+import monitoring.Metrics
 import org.joda.time.{DateTime, LocalDate}
 import play.api.libs.concurrent.Execution.Implicits._
-
 import scala.concurrent.Future
 
 class ScanamoAttributeService(client: AmazonDynamoDBAsync, table: String)
     extends AttributeService with LazyLogging {
 
+  val metrics = Metrics("ScanamoAttributeService") //referenced in CloudFormation
```

Comment:
my only suggestion would be to overload the name and make it more "id' like rather than a comment, e.g. aws-scanamoAttributeService and metric-oldDynamoItem rather than looking like a generic string description

Reply:
Yeah that sounds sensible - I guess we could also store these strings in our config files. Then we could group all of the metrics we care about (which makes it easy to delete spurious custom metrics with confidence). It would also probably make it a bit more obvious that changing them might break stuff.